### PR TITLE
sc-36 removed percentage from admin

### PR DESCRIPTION
**Pull request by adusmani17**

### CODE REVIEW

File: src/components/common/StatBox.jsx [hunk 1]
```diff
@@ -11,52 +11,61 @@ import PropTypes from "prop-types";
 
 import ProgressCircle from "./ProgressCircle";
 
-const StatBox = ({ title, subtitle, icon, progress, increase }) => {
-    const theme = useTheme();
-    const isMobile = useMediaQuery("(max-width:480px)");
+const StatBox = ({ title, subtitle, icon, progress, increase, role }) => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery("(max-width:480px)");
+  console.log(role,'sdfg')
```
Comment (ahmadaafaq) [inline src/components/common/StatBox.jsx:17]: remove console

File: src/components/dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -39,6 +39,7 @@ const Dashboard = ({ rolePriority = null }) => {
   const { typography } = themeSettings(theme.palette.mode);
   const dashboardAttributes = rolePriority === 1 ? ['student', 'school', 'teacher', 'employee']
     : rolePriority !== 1 ? ['student', 'bus', 'teacher', 'employee'] : null;
+    console.log(rolePriority, 'wwww')
```
Comment (ahmadaafaq) [inline src/components/dashboard/Dashboard.jsx:42]: remove console

**Review by ahmadaafaq — CHANGES_REQUESTED** [review]
left comments

File: src/components/common/StatBox.jsx (hunk 1 above)
Comment (adusmani17) [reply src/components/common/StatBox.jsx]: fixed

File: src/components/dashboard/Dashboard.jsx (hunk 2 above)
Comment (adusmani17) [reply src/components/dashboard/Dashboard.jsx]: fixed

**Comment by ahmadaafaq** [comment]
looks good

**Pull request merged**